packages/arb-validator-core/gobridge/executionChallenge.go

```diff
@@ -81,7 +83,7 @@ func (c *ExecutionChallenge) BisectAssertion(
 		logAccs[bisectionCount],
 	)
 
-	if !c.client.challenges[c.contractAddress].challengerDataHash.Equals(valprotocol.ExecutionDataHash(totalSteps, preconditionHash, assertionHash)) {
+	if !c.client.challenges[c.contractAddress].challengeData.challengerDataHash.Equals(valprotocol.ExecutionDataHash(totalSteps, preconditionHash, assertionHash)) {
```

Comment:
Unnecessary indirection to get `challengeData`

Reply:
fixed